clusterloader2/testing/l4ilb/config-nodepool-upgrade.yaml

```diff
@@ -0,0 +1,124 @@
+# Timer measurement is used to measure the time between the start of nodepool upgrade and when
+# the ILB services are running and reachable.
+# ServiceCreationLatency measurement is used only for determining the moment when all of the
+# ILBs are reachable. Since it reports time based on the initial creation of the service, and
+# not from the nodepool upgrade, the measured time will not be shown.
+
+# Constants
+{{$LB_REPLICAS_PER_NS := DefaultParam .CL2_LB_REPLICAS_PER_NS 5}}
+{{$LB_BACKEND_SIZE := DefaultParam .CL2_LB_BACKEND_SIZE 300}}
+{{$EXTERNAL_TRAFFIC_POLICY := DefaultParam .CL2_EXTERNAL_TRAFFIC_POLICY "Cluster"}}
+{{$ilbWaitTimeout := DefaultParam .CL2_ILB_WAIT_TIMEOUT "10m"}}
+{{$ilbQPS := DefaultParam .CL2_ILB_TEST_QPS 20}}
+{{$NODEPOOL_UPGRADE_LABEL := "ilb-np-upgrade"}}
+{{$namespaces := 1}}
+
+# Command to be executed
+{{$NODEPOOL_UPGRADE_COMMAND := DefaultParam .CL2_EXEC_COMMAND nil}}
```

Comment:
I wouldn't call this parameter (and also the test itself) as nodepool-upgrade. This test is more generic and can be run as both node-pool upgrade, but also any other arbitrary operation.

Reply:
Good point. Thank you.
I've made the changes and I will run another test to make sure it still works.